src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -1523,6 +1552,33 @@ public async Task<IActionResult> KitDetails(KitDetailsViewModel model)
             return View(model);
         }
 
+        [HttpPost]
+        public async Task<IActionResult> DeleteKitSelection(KitSelectionsViewModel model)
+        {
+            var settings = await _performerSchedulingService.GetSettingsAsync();
+            var schedulingStage = _performerSchedulingService.GetSchedulingStage(settings);
+            if (schedulingStage == PsSchedulingStage.Unavailable)
+            {
+                return RedirectToAction(nameof(PerformerManagementController.KitSelections), new { id = model.Kit.Id });
+            }
+
+            try
+            {
+                var branchSelection = await _performerSchedulingService.GetBranchProgramSelectionByIdAsync(model.BranchSelectionId);
+                var branch = await _siteService.GetBranchByIdAsync(branchSelection.BranchId);
+                var kit = await _performerSchedulingService.GetKitByIdAsync(model.Kit.Id);
+                branchSelection.IsDeleted = true;
+                await _performerSchedulingService.DeleteBranchSelectionAsync(branchSelection);
+                ShowAlertSuccess($"{branch.Name}'s selection of \"{kit.Name}\" is deleted!");
+            }
+            catch (GraException gex)
+            {
+                ShowAlertDanger("Unable to delete selection: ", gex);
+                return RedirectToAction(nameof(PerformerManagementController.KitSelections), new { id = model.Kit.Id });
```

Comment:
Again, shouldn't be needed since it does this on line 1579.

Reply:
Fixed in 5044ff45f01c4c5f7629dd13bae6f7f7b8c74a0a